lib/loader/index.js

```diff
@@ -233,30 +242,34 @@ function postInstantiate(extendedExports, instance) {
     return new Type(buffer, bufPtr, U32[bufPtr + SIZE_OFFSET >>> 2] >>> alignLog2);
   }
 
-  extendedExports.__getInt8Array = getTypedArray.bind(null, Int8Array, 0);
-  extendedExports.__getInt8ArrayView = getTypedArrayView.bind(null, Int8Array, 0);
-  extendedExports.__getUint8Array = getTypedArray.bind(null, Uint8Array, 0);
-  extendedExports.__getUint8ArrayView = getTypedArrayView.bind(null, Uint8Array, 0);
-  extendedExports.__getUint8ClampedArray = getTypedArray.bind(null, Uint8ClampedArray, 0);
-  extendedExports.__getUint8ClampedArrayView = getTypedArrayView.bind(null, Uint8ClampedArray, 0);
-  extendedExports.__getInt16Array = getTypedArray.bind(null, Int16Array, 1);
-  extendedExports.__getInt16ArrayView = getTypedArrayView.bind(null, Int16Array, 1);
-  extendedExports.__getUint16Array = getTypedArray.bind(null, Uint16Array, 1);
-  extendedExports.__getUint16ArrayView = getTypedArrayView.bind(null, Uint16Array, 1);
-  extendedExports.__getInt32Array = getTypedArray.bind(null, Int32Array, 2);
-  extendedExports.__getInt32ArrayView = getTypedArrayView.bind(null, Int32Array, 2);
-  extendedExports.__getUint32Array = getTypedArray.bind(null, Uint32Array, 2);
-  extendedExports.__getUint32ArrayView = getTypedArrayView.bind(null, Uint32Array, 2);
+  const ctors = [
```

Comment:
This var and `bigctors` seem unnecessary, can just iterate over the literal

Reply:
👍